plugins/Sid/SidInstrument.h

```diff
@@ -82,19 +84,20 @@ class SidInstrument : public Instrument
 {
 	Q_OBJECT
 public:
-	enum FilerType {
+	enum class FilterType {
 		HighPass = 0,
 		BandPass,
 		LowPass,
-		NumFilterTypes
+		Count
 	};
+	constexpr static auto NumFilterTypes = static_cast<std::size_t>(FilterType::Count);
 	
-	enum ChipModel {
+	enum class ChipModel {
 		sidMOS6581 = 0,
 		sidMOS8580,
```

Comment:
Should these be capitalized like the other enums?
Maybe `SID_MOS6581`? I think having an underscore would be fine in this case.

Reply:
We could probably drop the `sid` entirely - I don't think there's any chance of confusion as it's inside the `SidInstrument` class.

Reply:
That sounds good to me - `MOS6581` and `MOS8580`.